.github/workflows/release.yml

```diff
@@ -86,19 +86,27 @@ jobs:
           retention-days: 1
 
   linux:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
+    container: ubuntu:18.04
```

Comment:
What is this for?

Reply:
So it still builds against ubuntu 18.04 but doesn't depend on ubuntu-18.04 github runners

Reply:
The reason we needed 18.04 was because of the GLIBC version. We can probably just use latest?

Reply:
Sure, but what about https://github.com/lapce/lapce/issues/134 then?
Maybe I should add a build for each Ubuntu LTS from 18.04 to 22.04 and we can upload artefacts for all three (18.04/20.04/22.04)?